Core/DetectorConstruction.cpp

```diff
@@ -164,7 +167,19 @@ void DetectorConstruction::ReloadMaterials(const G4String& material)
     fPlexiglass->FillIntensities();
   } else if (material == "pa6") {
     fSmallChamberRun7Material->ChangeMaterialConstants();
-    fSmallChamberRun7Material->FillIntensities();
+    fSmallChamberRun7Material->FillIntensities(); 
+  } else if (material == "StainlessSteel"){
```

Comment:
could you use stainlessSteel and siliconDioxide names, to preseve a convention that material are starting with small letter? 
[For me to remember: add names of the materials to the manual/parameters etc.]